x/clp/client/cli/tx.go

```diff
@@ -38,7 +38,7 @@ func GetTxCmd(cdc *codec.Codec) *cobra.Command {
 
 func GetCmdCreatePool(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-pool",
+		Use:   "create-pool --from [key] --symbol [assert-symbol] --nativeAmount [amount] --externalAmount [amount]",
```

Comment:
I think there's a typo here

Reply:
where??

Reply:
@utx0 you have `assert-symbol` when it should be `asset-symbol`